modules/.vale.ini

```diff
@@ -5,7 +5,10 @@ MinAlertLevel = suggestion
 Vocab = OpenShiftDocs
 
 [[!.]*.adoc]
-BasedOnStyles = OpenShiftAsciiDoc, AsciiDoc, RedHat
+BasedOnStyles = OpenShiftAsciiDoc, AsciiDoc, RedHat, https://github.com/jhradilek/asciidoctor-dita-vale/releases/latest/download/AsciiDocDITA.zip
```

Comment:
I don't think that goes here on the BasedOnStyles line. That line went onto the Packages line

Reply:
oh interesting, it may inherit the packages from the root then. i'll remove this